spec/unit/puppet/provider/firewalld_ipset_spec.rb

```diff
@@ -15,6 +15,13 @@
   let(:provider) { resource.provider }
 
   before do
+    tempfile = stub('tempfile', class: Tempfile,
+                                write: true,
+                                flush: true,
+                                close!: true,
+                                close: true,
+                                path: '/tmp/ipset-rspec')
```

Comment:
Reading from /tmp is insecure and could introduce a race condition. Suggest using a dedicated directory that is root:root 0700 and outside of temp.

Reply:
In the test ?

Reply:
not sure I need to notify using @ghoneycutt , sorry for the spam

Reply:
Why not just use a real tempfile?

This is going to be an issue if a system is trying to run multiple tests at the same time.

A `before(:each)` with an instance variable should be fine (just tell rubocop to shut up).